src/pages/feed/index.vue

```diff
@@ -384,12 +440,40 @@ export default {
       this.fetchNFTDisplayStateListByAddress(this.getAddress);
       this.updateTopRankedCreators();
     }
+    this.addInfiniteScrollListener();
+  },
+  beforeDestroy() {
+    this.removeInfiniteScrollListener();
   },
   methods: {
-    ...mapActions(['lazyGetUserInfoByAddress']),
+    ...mapActions([
+      'lazyGetUserInfoByAddress',
+      'lazyGetUserInfoByAddresses',
+      'lazyFetchEventsMemo',
+      'lazyGetNFTClassMetadata',
+      'fetchFolloweeWalletEvent',
+    ]),
     fetchUserInfo() {
       this.lazyGetUserInfoByAddress(this.getAddress);
     },
+    async batchFetchMemo() {
+      this.allowFetch = false;
+      const currentEventToFetch = Math.min(
+        this.eventsToFetch,
+        this.pendingMemoFetchList.length
+      );
+      const fetchList = this.pendingMemoFetchList.slice(0, currentEventToFetch);
+
+      const fetchPromises = fetchList.map(event =>
+        this.lazyFetchEventsMemo(event)
```

Comment:
should we wrap a try-catch in case one of them fail causing entire promise to throw error?